python/eos/pyhf_likelihood.py

```diff
@@ -79,7 +108,12 @@ def factory(cache, workspace, parameter_map={}):
     def priors(self):
         priors = []
         for name, _ in self.model.config.modifiers:
-            param_name = self.parameter_map[name] if name in self.parameter_map else 'pyhf::' + name
+            param_name = 'pyhf::' + name
+            if name in self.parameter_map:
+                if isinstance(self.parameter_map[name], str):
+                    param_name = self.parameter_map[name]
+                elif isinstance(self.parameter_map[name], dict):
+                    continue
```

Comment:
I assume now that if a dictionary is passed, it is an observable and hence no prior is assigned. It is the desired behaviour, but maybe not the best way to implement it.